.github/workflows/examples.yml

```diff
@@ -0,0 +1,106 @@
+name: Examples
+
+on:
+  push:
+    branches:
+      - "master"
+      - "maintenance/.+"
+  pull_request:
+    branches:
+      - "master"
+      - "maintenance/.+"
+  schedule:
+    # Nightly tests run on master by default:
+    #   Scheduled workflows run on the latest commit on the default or base branch.
+    #   (from https://help.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule)
+    - cron: "0 0 * * *"
+
+jobs:
+  test:
+    if: (github.event_name == 'schedule' && github.repository == 'openforcefield/openff-toolkit') || (github.event_name != 'schedule')
+    name: Test on ${{ matrix.os }}, Python ${{ matrix.cfg.python-version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+        cfg:
+          - python-version: 3.7
+
```

Comment:
Per our discussion today, let's expand this testing matrix to test {rdkit-only, openeye-only}. The way we do that in the standard CI is [here](https://github.com/openforcefield/openff-toolkit/blob/7c3dfbaed9cda911b70065626414c7e382bf9ff9/.github/workflows/CI.yml#L73-L88)